src/lib/style.store.ts

```diff
@@ -9,11 +9,12 @@ export const logoSchema = z
     width: z.number().optional(),
   })
   .strict();
-
+  
 export const styleSchema = z
   .object({
     checksAndRadios: z.union([z.literal('animated'), z.literal('standard')]).optional(),
     labels: z.union([z.literal('floating').optional(), z.literal('stacked')]).optional(),
+    showPassword: z.union([z.string(), z.null()]).optional(),
```

Comment:
Let's make sure the type is as narrow as possible. What I'd recommend is using `z.literal` rather than the generic `z.string`. Also, I don't think it needs `z.null` if it's optional. The fact that it's optional should allow a _nullish_ value.

@ryanbas21 Can you double check this for me?

Reply:
Also, I'd like to have `'none'` as an option for the customer, in addition to `'button'`, `'checkbox'` and `undefined`. Can we add this as a possible value, so they can essentially disable the show password feature entirely?

@ryanbas21 Thoughts?

Reply:
I agree. I don't think the code as is makes a ton of sense.


a union of literals would be more fitting if possible.

So we would have 

```
z.union(z.literal('none'), z.literal('button'), z.literal('checkbox'))
```

Reply:
done